dissect/target/plugins/os/windows/lnk.py

```diff
@@ -168,7 +168,18 @@ def lnk(self, directory: Optional[str] = None) -> Iterator[LnkRecord]:
 
     def lnk_entries(self, path: Optional[str] = None) -> Iterator[TargetPath]:
```

Comment:
I believe this typehint is not correct anymore. A list of strings can be passed now? The same is true for the exported `lnk()` function.

Reply:
Could you also change the help for the `--directory` argument to reflect these changes?

Reply:
It is still correct, because the user can still only provide a single path. However, it will do an recursive glob whenever a directory was provided by the user, which results in multiple `TargetPath` objects to be returned.